src/edu/csus/ecs/pc2/ui/ScoringPropertiesPane.java

```diff
@@ -115,7 +121,8 @@ protected void refreshProperties() {
             propertyListBox.addRow(objects);
         }
         
-        propertyListBox.autoSizeAllColumns();
+        propertyListBox.setColumnSize(0,this.lengthOfColumns);
```

Comment:
Same as above.  Why are you using "this." ?

Reply:
I think I pickup that habit in Python. Java implicitly adds preprends that i think so as you suggest, there is no need for that here.